doc/sphinx/source/recipes/recipe_ref.rst

```diff
@@ -23,6 +23,12 @@ Recipes are stored in recipes
   Calculate transient climate response to cumulative CO2 emissions (TCRE)
 * ref/recipe_ref_cre.yml:
   Maps and zonal means of longwave and shortwave cloud radiative effect
+* ref/recipe_model_benchmarking_timeseries_region.yml:
+  Time series in comparison with reference data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_timeseries.yml <recipe_benchmarking>`
+* ref/recipe_model_benchmarking_annual_cycle_region.yml:
+  Annual cycle in comparison with reference data for a region defined through a shape file, based on :ref:`recipe_model_benchmarking_annual_cycle.yml <recipe_benchmarking>`
+* ref/recipe_seaborn_trend_regions.yml:
+  Linear Trends for all IPCC regions compared with reference data, based on :ref:`recipe_seaborn.yml <recipes_seaborn_diag>`
```

Comment:
Here I would just use one region, as you do for the other two recipes. Then you can adjust the text to be as general as the texts in the description of the other recipes before.